docs/Getting Started/Adding the Lock Screen/index.md

```diff
@@ -133,20 +133,27 @@ lockScreenConfig.setCustomView(customViewInt);
 !@
 
 ## Customizing the Lock Screen State
+In SDL iOS v6.4 a new parameter `displayMode` has been added to the `SDLLockScreenConfiguration` to control the state of the lock screen and the older boolean parameters have been deprecated.
+
+| DisplayMode | Description |
```

Comment:
Please arrange this table better with white-spacing. Look at other tables or ask if you need more information.